.github/scripts/check_degradation.sh

```diff
@@ -0,0 +1,23 @@
+current_branch=$(git rev-parse --abbrev-ref HEAD) 
+critcmp new_branch main_branch | awk 'NR>2 {
+    item = $1
+    before = $3
+    after = $7
+    before_val = ($3 ~ /ns/) ? $3 : ($3 ~ /µs/) ? $3 * 1000 : ($3 ~ /ms/) ? $3 * 1000000 : "invalid"
+    after_val = ($7 ~ /ns/) ? $7 : ($7 ~ /µs/) ? $7 * 1000 : ($7 ~ /ms/) ? $7 * 1000000 : "invalid"
```

Comment:
I think the critcmp is already showing the relative comparison between measurements

![image](https://github.com/tailcallhq/tailcall/assets/8974488/aa08ca44-488d-4600-bded-783ae94c6ad6)

Can we use them instead of calculating it?

Reply:
thanks for pointing it out i never noticed this

Reply:
@meskill by calculating the percentage change will give us more accurate results .
as the minimum change in percentage in the above is 1 . What do you think